src/features/TodoList/TodoListItem.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import TextInputWithLabel from '../../assets/shared/TextInputWithLabel';
+
+function TodoListItem({ item, onCompleteTodo, onUpdateTodo }) {
+  const [isCompleted, setIsCompleted] = useState(false);
+  const [isEditing, setIsEditing] = useState(false);
+  const [workingTitle, setWorkingTitle] = useState(item.title);
+
+  const handleCheckboxCheck = (itemId) => {
```

Comment:
Good instinct on having a descriptive function name. It's also a good id to make it as concise as possible. `handleCheckBox` works just fine.